### DISCUSSION TITLE
How do I use a custom template repo in vscode?

### DISCUSSION THREAD

**Discussion by tclarke** [devcontainers/community#101] [General]
I've got template and feature repos internal to my org that I'd like to use. I can use `devcontainer templates apply` with no problem but I can't figure out how to get my templates to show up in vscode in the Add Dev Container Config Files wizard.

**Comment by samruddhikhandale** [comment]
Hi 👋 

The only way to add Features or Templates to the wizard is to make a PR against the [index](https://github.com/devcontainers/devcontainers.github.io/blob/gh-pages/_data/collection-index.yml). If it's private-use templates, there's no associated UI to apply (private index is something that would be great to do, we discussed this recently but haven't implemented yet) 

Feel free to use https://github.com/devcontainers/spec/issues/71#issuecomment-1415959648 for tracking the request
